app/views/calcs/new.html.haml

```diff
@@ -1,36 +1,66 @@
-= form_for :calc, url:calcs_path,method: :post do |f|
-  =f.label 'Cpu'
-  %br
-  =f.select :cpu_id, Cpu.all.collect {|p| [ (Company.find(p.company_id).company_name+' '+Brand.find(p.brand_id).brand_name+' Socket '+SocketCpu.find(p.socket_cpu_id).socket_name+' '+p.name).to_s, p.id ] }, { include_blank: true },id:'ec1'
-  %br
-  =f.label 'Vga'
-  %br
-  =f.select :vga_id, Vga.all.collect {|p| [ (Company.find(p.company_id).company_name+' '+Brand.find(p.brand_id).brand_name+' '+p.vga_name).to_s, p.id ] }, { include_blank: true },id:'ec2'
-  %br
-  =f.label 'Vga count'
-  %br
-  =f.select :vga_count, (1..4).to_a {|p| [ 'x', p ] }, { include_blank: true },id:'ec6'
-  %br
-  =f.label 'Ram'
-  %br
-  =f.select :ram_id, Ram.all.collect {|p| [ Brand.find(p.brand_id).brand_name, p.id ] }, { include_blank: true },id:'ec3'
-  %br
-  =f.label 'Ram count'
-  %br
-  =f.select :ram_count, (1..8).to_a {|p| [ 'x', p ] }, { include_blank: true },id:'ec4'
-  %br
-  =f.label 'Fan count'
-  %br
-  =f.select :fan_count, (1..10).to_a {|p| [ 'x', p ] }, { include_blank: true },id:'ec5'
-  %br
-  =f.label 'Water cooling'
-  %br
-  =f.select :water_cooling_id, WaterCooling.all.collect {|p| [ 'Pomp '+p.pomp_count.to_s+'x'+', Fan'+p.fan_count.to_s+'x'+p.id.to_s, p.id ] }, { include_blank: true },id:'ec6'
-  =f.select :socket, SocketCpu.all.collect {|p| [ p.socket_name, p.id ] }, { include_blank: true },id:'ec7'
-  =f.submit 'calc',url:calcs_path
+.div
+  .row
+    .col-md-6
+      = form_for @calc, method: :post do |f|
+        - if @calc.errors.any?
+          %div#error_explanation
+            %h2
+              = pluralize(@calc.errors.count, "error")
```

Comment:
.alert.alert-danger
    %strong Please fix this errors
    = pluralize...........